README.md

```diff
@@ -131,142 +131,174 @@ Netdata is built around a **modular metrics processing pipeline**.
   - Netdata is a popular open-source project and is frequently tested by many security analysts.
-  - Check also our [security policies and advisories published so far](https://github.com/netdata/netdata/security).
-
-### :cyclone: Will this consume a lot of resources on my servers?
+  - Check also our <a href="https://github.com/netdata/netdata/security">security policies and advisories published so far</a>.
+<br><br>
+</details>
 
+<details>
+<summary>:cyclone: Will this consume a lot of resources on my servers?</summary>
+<br>
 No. It will not! We promise this will be fast!
-
+<br><br>
 Although each Netdata Agent is a complete monitoring solution packed into a single application, and despite the fact that Netdata collects **every metric every single second** and trains **multiple ML models** per metric, you will find that Netdata has amazing performance! In many cases, it outperforms other monitoring solutions that have significantly fewer features or far smaller data collection rates.
-
+<br><br>
 This is what you should expect:
-
+<br><br>
   - For production systems, each Netdata Agent with default settings (everything enabled, ML, Health, DB) should consume about 5% CPU utilization of one core and about 150 MiB or RAM. By using a Netdata parent and streaming all metrics to that parent, you can disable ML & health and use an ephemeral DB mode (like `alloc`) on the children, leading to utilization of about 1% CPU of a single core and 100 MiB of RAM. Of course, these depend on how many metrics are collected.
   - For Netdata Parents, for about 1 to 2 million metrics, all collected every second, we suggest a server with 16 cores and 32GB RAM. Less than half of it will be used for data collection and ML. The rest will be available for queries.
-
+<br><br>
 Netdata has extensive internal instrumentation to help us reveal how the resources consumed are used. All these are available in the "Netdata Monitoring" section of the dashboard. Depending on your use case, there are many options to optimize resource consumption.
-
+<br><br>
 Even if you need to run Netdata on extremely weak embedded or IoT systems, you will find that Netdata can be tuned to be very performant.
+<br><br>
+</details>
 
-### :scroll: How much retention can I have?
-
+<details>
+<summary>:scroll: How much retention can I have?</summary>
+<br>
 As much as you need!
-
+<br><br>
 Netdata supports **tiering**, to downsample past data and save disk space. With default settings, it has 3 tiers:
-
-  1. `tier 0`, with high resolution, per-second, data.
-  2. `tier 1`, mid-resolution, per minute, data.
-  3. `tier 2`, low-resolution, per hour, data.
-
+<br><br>
+<ol>
+  <li><code>tier 0</code>, with high resolution, per-second, data.</li>
+  <li><code>tier 1</code>, mid-resolution, per minute, data.</li>
+  <li><code>tier 2</code>, low-resolution, per hour, data.</li>
+</ol>
+<br>
```

Comment:
Why do we need to use HTML tags for line breaks and lists in a markdown file?

Reply:
Unfortunately if want to put content inside a details tag so it's collapsible then it needs to all be html inside the details tag. 

A bit annoying but I think worth it to make the FAQ easier to scan.

Reply:
I know that it needs to be inside the details tag. The question is why do you use HTML tags (`<code>`, `<br>`, `<li>`) instead of markdown syntax?

Reply:
Using markdown in there was not getting rendered properly so I had to use html tags instead

Reply:
What exactly wasn't getting rendered properly? I am asking because I have never had such problem

See:

- [rendered version of the same section](https://github.com/ilyam8/playground#playground)
- [raw version of the same section](https://raw.githubusercontent.com/ilyam8/playground/main/README.md)

Reply:
hmm you are right not sure why or what was going on that i thought that. 

removed all [here](https://github.com/netdata/netdata/pull/15418/commits/774ba32ff71c79f0d57dd7a3f40ad2501b44e8ab) apart from just a [<br> at start](https://github.com/netdata/netdata/pull/15418/commits/c4b23c4be16b91141aedcb7ffef53e3c6c0b8991) to give some whitespace after the question line - could not find a md way to do that.

Reply:
actually now i'm still missing some newlines for some reason i tried "  " at end but not working that's where "<br><br>" was working before.